### ISSUE TITLE
How to train with batch size 1

### ISSUE THREAD

**Issue by ptran1203** [chongruo/detectron2-ResNeSt#60]
I try to train an instance segmentation model with resnest200 backbone with image size 1024. But due to memory constraint, I can fit only batch size of 1. But when I use SyncBN, It requires more than 1 sample.  Can I use other batch norm (or set norm to None) without effect performance
